src/MLJLIBSVMInterface.jl

```diff
@@ -549,4 +553,859 @@ MMI.target_scitype(::Type{<:Union{NuSVR, EpsilonSVR}}) =
+$DOC_KERNEL
+
+- `cost=1.0` (range (0, `Inf`)): the parameter denoted ``C`` in the
+  cited reference; for greater regularization, decrease `cost`
+
+- `epsilon=0.1` (range (0, `Inf`)): the parameter denoted ``ϵ`` in the
+  cited reference; `epsilon` is the thickness of the "penalty-free"
+  neighborhood of the decision surface.
+
+- `cachesize=200.0` cache memory size in MB
+
+- `tolerance=0.001`: tolerance for the stopping criterion
+
+- `shrinking=true`: whether to use shrinking heuristics
+
+
+# Operations
+
+- `predict(mach, Xnew)`: return predictions of the target given
+  features `Xnew` having the same scitype as `X` above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `libsvm_model`: the trained model object created by the LIBSVM.jl package
+
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `gamma`: actual value of the kernel parameter `gamma` used in training
+
+
+# Examples
+
+## Using a built-in kernel
+
+```
+using MLJ
+import LIBSVM
+
+EpsilonSVR = @load EpsilonSVR pkg=LIBSVM            # model type
+model = EpsilonSVR(kernel=LIBSVM.Kernel.Polynomial) # instance
+
+X, y = make_regression(rng=123) # table, vector
+mach = machine(model, X, y) |> fit!
+
+Xnew, _ = make_regression(3, rng=123)
+
+julia> yhat = predict(mach, Xnew)
+3-element Vector{Float64}:
+  0.2512132502584155
+  0.007340201523624579
+ -0.2482949812264707
+```
+
+## User-defined kernels
+
+```
+k(x1, x2) = x1'*x2 # equivalent to `LIBSVM.Kernel.Linear`
+model = EpsilonSVR(kernel=k)
+mach = machine(model, X, y) |> fit!
+
+julia> yhat = predict(mach, Xnew)
+3-element Vector{Float64}:
+  1.1121225361666656
+  0.04667702229741916
+ -0.6958148424680672
+```
+
+See also [`NuSVR`](@ref),
+[LIVSVM.jl](https://github.com/JuliaML/LIBSVM.jl) and the
+[documentation](https://github.com/cjlin1/libsvm/blob/master/README)
+for the original C implementation.
+
+"""
+EpsilonSVR
+
+
+# ## NuSVR
+
+"""
+$(MMI.doc_header(NuSVR))
+
+$DOC_ALGORITHM
+
+This model is a re-parameterization of `EpsilonSVR` in which the
+`epsilon` hyper-parameter is replaced with a new parameter `nu`
+(denoted ``ν`` in the cited reference) which attempts to control the
+number of support vectors directly.
+
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with:
+
+    mach = machine(model, X, y)
+
+where
+
+- `X`: any table of input features (eg, a `DataFrame`) whose columns
+  each have `Continuous` element scitype; check column scitypes with
+  `schema(X)`
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Continuous`; check the scitype with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+
+# Hyper-parameters
+
+- $DOC_KERNEL
+
+- `cost=1.0` (range (0, `Inf`)): the parameter denoted ``C`` in the
+  cited reference; for greater regularization, decrease `cost`
+
+- `nu=0.5` (range (0, 1]): An upper bound on the fraction of margin
+  errors and a lower bound of the fraction of support vectors. Denoted
```

Comment:
Instead of `margin error`, sklearn uses `training error` here. Schölkopf et al. use simply say `fraction of errors`, with errors defined as `training points lying outside the tube or slab`. I'm not well versed in SVR, much less nu-SVR, so I'm not sure how standard each of these terms are.

Reply:
Hows this?

- `nu=0.5` (range (0, 1]): An upper bound on the fraction of training
  errors and a lower bound of the fraction of support vectors.
  Denoted ``ν`` in the cited paper. Changing `nu` changes the
  thickness of some neighborhood of the graph of the prediction
  function (called a "tube" or "slab") and a training error is said to
  occur when a data point `(x, y)` lies outside of that neighborhood.